### PR TITLE
 Adds in tables listing all of the supported and unsupported OpCodes

### LINKED ISSUE #12: Add in exhaustive list of allowed OpCodes
I have a spreadsheet with all of the OpCodes that are explicitly allowed/disallowed for WebGPU. This information should appear in an appendix

### PR DESCRIPTION
**Pull request by zoddicus**
Fixes #12

### CODE REVIEW

File: execution-env.md [hunk 1]
```diff
@@ -570,6 +570,7 @@ Fusing and reassociation of floating point operations is allowed when those inst
 
 *   OpUndef is not allowed.
 *   OpVectorShuffle may not have a component literal with value 0xFFFFFFFF.
+*   Supported and unsupported OpCodes are iterated in [Appendix A](#a-supported-opcodes) and [Appendix B](#b-unsupported-opcodes).
```
Comment (dneto0) [inline execution-env.md]: We should not list the _unsupported_ opcodes.  Just keep it to the whitelist.
Also, I would not make this a bullet point.  I would just pull it out as its own paragraph.

File: execution-env.md [hunk 2]
```diff
@@ -722,3 +723,2275 @@ Note: The **std430** in GLSL satisfies these rules.
...
+  </tr>
+  <tr>
+   <td>332
+   </td>
+   <td>OpDecorateId
+   </td>
+  </tr>
+</table>
+
+## B: Unsupported OpCodes
```
Comment (dneto0) [inline execution-env.md]: Please remove this appendix.

**Review by dneto0 — CHANGES_REQUESTED** [review]

File: execution-env.md (hunk 1 above)
Comment (zoddicus) [reply execution-env.md]: Done

File: execution-env.md (hunk 2 above)
Comment (zoddicus) [reply execution-env.md]: Done

**Comment by zoddicus** [comment]
PTAL, I have made the requested changes.

**Comment by zoddicus** [comment]
@dneto0 Are there any other changes that are needed?

File: execution-env.md [hunk 3]
```diff
@@ -722,3 +724,1149 @@ Note: The **std430** in GLSL satisfies these rules.
...
+  <tr>
+   <td>100
+   </td>
+   <td>OpImage
+   </td>
+  </tr>
+  <tr>
+   <td>105
+   </td>
+   <td>OpImageQueryLod
```
Comment (dneto0) [inline execution-env.md:1169]: ImageQuery capability is allowed, and that also permits the following instructions:

- OpImageQuerySizeLod
- OpImageQuerySize
- OpImageQueryLevels
- OpImageQuerySamples

**Review by dneto0 — CHANGES_REQUESTED** [review]
A close look says a handful of instructions are missing, enabled by the ImageQuery capability.

(If an instruction is enabled by multiple capabilities, then it's an "or": the instruction is enabled if *any* of those capabilities is declared.)

File: execution-env.md (hunk 3 above)
Comment (zoddicus) [reply execution-env.md:1169]: Done. I have manually merged them here and will have to look at handling this correctly in my spreadsheet.

**Review by dneto0 — APPROVED** [review]

**Comment by kenrussell** [comment]
Super work Ryan! Thanks for putting this together!

As an aside - isn't there more terse Markdown syntax for tables? If so, if at all possible, could this be reformatted using it? Thanks in advance.

**Comment by zoddicus** [comment]
Looks like there is. I have created #13 to track converting, it should be pretty easy.

**Pull request merged**